lib/webhooks/webhooks.ex

```diff
@@ -23,12 +24,18 @@ defmodule ApplicationRunner.Webhooks.Webhook do
   def changeset(webhook, params \\ %{}) do
     webhook
     |> cast(params, [:action, :props])
-    |> validate_required([:action])
+    |> validate_required([:environment_id, :action])
     |> foreign_key_constraint(:environment_id)
+    |> foreign_key_constraint(:user_id)
   end
 
   def new(env_id, params) do
     %Webhook{environment_id: env_id}
     |> Webhook.changeset(params)
   end
+
+  def new(env_id, user_id, params) do
```

Comment:
@pichoemr Is it better to have two `new` methods ? One with the env_id and one with the env_id + user_id ?

Or is it better to have only one `new` method only with the required `env_id` parameter and pass the `user_id` in the `params` (and cast it in the changeset) ?